core/modules/widgets/macrocall.js

```diff
@@ -55,9 +55,18 @@ MacroCallWidget.prototype.execute = function() {
 	// Are we rendering to HTML?
 	if(this.renderOutput === "text/html") {
 		// If so we'll return the parsed macro
-		var parser = this.wiki.parseText(this.parseType,text,
-							{parseAsInline: !this.parseTreeNode.isBlock});
-		parseTreeNodes = parser ? parser.tree : [];
+		// Check if we've already cached parsing this macro
+		var parser;
+		if(variableInfo.srcVariable && variableInfo.srcVariable.parser) {
+			parser = variableInfo.srcVariable.parser;
+		} else {
+			parser = this.wiki.parseText(this.parseType,text,
+								{parseAsInline: !this.parseTreeNode.isBlock});
+			if(variableInfo.isCacheable && variableInfo.srcVariable) {
+				// variableInfo.srcVariable.parser = parser;
```

Comment:
@Jermolene .. Just found this line. Is it intended, to be a comment?

Reply:
@Jermolene ... I'm still not sure, if this comment is intended?

Reply:
I think that must be an outdated diff, the current version at https://github.com/Jermolene/TiddlyWiki5/blob/master/core/modules/widgets/macrocall.js doesn't have it